spec/controllers/aspect_memberships_controller_spec.rb

```diff
@@ -1,12 +1,13 @@
-#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
+##   Copyright (c) 2010-2011, Diaspora Inc.  This file is   Copyright (c) 2010-2011, Diaspora Inc.  This file is
 #   licensed under the Affero General Public License version 3 or later.  See
 #   the COPYRIGHT file.
 
 require 'spec_helper'
+require 'pry-byebug'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.